proposals/dsp-004.md

```diff
@@ -0,0 +1,31 @@
+# Recruitment Incentives
+
+**Author(s)**: @ronboger, @orishim
+
+## Summary
+
+Recruitment referral model that ties incentive to revenue generated by the recruited builder. 
+
+## Why should we fund this?
+
+For dOrg to continue to provide high quality service and take on exciting projects, the organization needs to be able to attract and maintain top engineering talent.
+
+## Anything else you'd like to add?
+
+The incentive will be equivalent to 5% of the recruited builder's first $50k earned from client work. dOrg makes 10 cents on every 80 cents earned by a builder on client work. In the case of a recruited builder, dOrg would effectively be splitting this 10 cents with the recruiter.
```

Comment:
I'd bring it up to 100-200k earned from client work. 2.5k is nice but the nature of even getting to it is uncertain. I'd also change it to "referrer" instead of "recruiter". Typical recruitment fee is 20% of first year salary (equivalently 20K in the case of an average builder), typical referral bonuses at companies vary in the 1-10K range

Reply:
I updated to 100k so that max reward is 5k, and also changed all "recruitment" language to "referral".